buildspec.yml

```diff
@@ -6,7 +6,7 @@ phases:
       nodejs: 12
       python: 3.8
     commands:
-      - n 16  # Force Node update to v16
+      - n 20  # Force Node update to v16
```

Comment:
@pauline2k trivial, but this comment should be updated to match.

Reply:
@lyttam right! PR #1431 consolidates.